maintainer/walberla_kernels/generate_lb_kernels.py

```diff
@@ -318,6 +318,21 @@ def patch_boundary_kernel(content, target_suffix):
             assert pop in content
         return content
 
+    def patch_boundary_force_accessor(content):
+        # make ids accessable from outside
+        content = content.replace("BlockDataID forceVectorID;", "")
+        content = content.replace(
+            "BlockDataID pdfsID;", "BlockDataID pdfsID;\nBlockDataID forceVectorID;\n")
+        content = content.replace("BlockDataID indexVectorID;", "")
+        content = content.replace(
+            "BlockDataID pdfsID;", "BlockDataID pdfsID;\nBlockDataID indexVectorID;")
```

Comment:
Can this be replaced by overloading static method `additional_member_variable()` in lbmpy_espresso with a function that injects a multi-line string containing `public:` followed by several getters? See e.g. the output of `print(ubb_data_handler.additional_member_variable)`, to be added after line 302.